File: testprojects/csharp/benchmark/domain/facebook/FacebookException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharpBenchmark.domain.facebook
{
    public class FacebookException : Exception
    {
        //Functional requirement 3.1.2
        //Test case 88: Only class presentation.gui.facebook.WallGUI may have a dependency with domain.facebook.FacebookException
        //Result: FALSE
        public FacebookException(string message) : base (message)
        {
 
        }
    }
}

File: testresources/java/benchmark_2012/domain/facebook/FacebookException.java
package domain.facebook;

//Functional requirement 3.1.2
//Test case 88: Only class presentation.gui.facebook.WallGUI may have a dependency with domain.facebook.FacebookException
//Result: FALSE
public class FacebookException extends Exception {
	public FacebookException(String message){
		super(message);
	}
}